includes/data-stores/class-wc-admin-reports-products-data-store.php

```diff
@@ -128,6 +116,32 @@ protected function get_order_by_sql_params( $query_args ) {
 		return $sql_query;
 	}
 
+	/**
+	 * Fills outer FROM clause of SQL request based on user supplied parameters.
+	 *
+	 * @param array  $query_args Parameters supplied by the user.
+	 * @param string $arg_name   Name of the FROM sql param.
+	 * @param string $id_cell    ID cell identifier, like `table_name.id_column_name`.
+	 * @return array
+	 */
+	protected function get_outer_from_sql_params( $query_args, $arg_name, $id_cell ) {
```

Comment:
Optional:  Should we change this name to `get_from_sql_params` since it looks like it can fill both from and outer from clauses now?

Reply:
Done!

fa2694deff7e2fb24a00e50d6c0f6b1293763d21